static/js/zamboni/buttons.js

```diff
@@ -179,16 +178,28 @@ var installButton = function() {
         };
     });
 
+    var showDownloadAnyway = function($button) {
+        var $installShell = $button.parents('.install-shell');
+        var $downloadAnyway = $installShell.find('.download-anyway');
+        if ($downloadAnyway.length) {
+            var href = $button.filter(':visible').attr('href');
+            var pos = href.lastIndexOf('/');
+            href = escape_(href.slice(0, pos) + '/type:attachment/' + href.slice(pos + 1));
```

Comment:
Yeah, it's broken in that scenario, so I need to think of a better way... I originally developed this thinking it would be using `downloads.file`, which does not have the platform bit, but since it's using `downloads.latest` which does, I need to rehink this... Maybe I can reverse the URL with `'type': 'attachment'` set and pass it all the way down to the JavaScript.